libraries/AP_GPS/AP_GPS_NMEA.cpp

```diff
@@ -252,6 +256,7 @@ bool AP_GPS_NMEA::_term_complete()
                     state.ground_speed     = _new_speed*0.01f;
                     state.ground_course    = wrap_360(_new_course*0.01f);
                     make_gps_time(_new_date, _new_time * 10);
+                    set_uart_timestamp(_sentence_length);
```

Comment:
This should be done in GGA sentences as well.

Reply:
GGA doesn't set state.last_gps_time_ms, so that would make it inconsistent I think